Source/WebCore/platform/video-codecs/BitReader.h

```diff
@@ -40,7 +40,16 @@ class BitReader {
     
     std::optional<uint64_t> read(size_t);
     std::optional<bool> readBit();
-    
+    template <typename T> std::optional<T> read()
+    {
+        static_assert(std::is_unsigned<T>::value);
+        auto value = read(sizeof(T) * 8);
```

Comment:
Maybe we should add readBytes?

Reply:
I dropped that idea, a bit weird to have a `readByte` that would here need to return a `typename T` which is up to 64 bits.